CODE_REVIEW.md

```diff
@@ -0,0 +1,73 @@
+# Code review requiements
+
+This document describe responsibilities of code reviewers when reviewing PRs.
+
+Pull request process is described in [contributing guide](./CONTRIBUTING.md#submitting-pull-requests)
+
+## Base checklist
+
+- All automated test steps pass (e.g. tests, lints, build)
+- PR title follows [commit conventions](CONTRIBUTING.md#commit-conventions)
+    - If PR format is different, reviewer should change it to follow the conventions
+- PR has a description with reasoning and change overview
+    - If description is missing but clear for reviewer, reviewer may add a comment or edit description
+- New feature or fix has tests proving it works
+    - Reviewer should request changes from contributor to add tests
+- If PR changes documented behaviour, there should be update to `/docs` or inline comments
+    - Reviewer should request changes from contributor to update docs
+- If PR introduces breaking changes, fixes a bug or adds a new feature, there should be a [release note](#release-notes)
+    - Reviewer may request changes from the contributor to add a release note
+    - Reviewer may add a release note by themself in order to unblock the merge process
+
+## Requesting changes
+
+It's recommended to request changes by submitting `comment` type reviews.
```

Comment:
Should we clarify that it doesn't mean that `Request changes` type of reviews should not be submitted. If someone thinks that some thing must be changed before the PR is merged, they should not hesitate adding `Request changes` type reviews.

Reply:
The issue with that is that person becomes a bottleneck for merging as mentioned below. That's why we should discourage it.